tests/unit/apps/test_auxiliary.py

```diff
@@ -870,10 +870,92 @@ def test_ovn_no_compatible_os_release(channel, model):
         )
 
 
+@patch("cou.apps.auxiliary.logger")
+def test_ovn_check_version_pinning_no_version_pinning_config(mock_logger, model):
```

Comment:
Can you join this and next steps with parametrize and `config` as argument, and I'll suggest removing mocking for logger. Logging is only informative, and I do not see reason to check if it was called.